README.md

```diff
@@ -401,6 +416,12 @@ appraisal bundle install
 appraisal bundle exec rspec
 ```
 
+To run the test suite faster (about 4x), you can skip the RuboCop invocations for testing
```

Comment:
I'm not a haml-lint maintainer, I'm just some guy, so take this with a grain of salt.

Given that [tests take 9s to run](https://github.com/sds/haml-lint/actions/runs/5248634051/jobs/9480384913?pr=346), I am not sure it's worth the extra complexity of this feature.

Reply:
It's a simple feature that has no code in the lib (It's basically a single stub in the tests). For me, it's the difference 17s and 5s. It can matter when working locally.

So for now I'll leave that in, but thanks for your input!